ts/parsers/elementparser.ts

```diff
@@ -127,6 +127,8 @@ class PowerpointElementParser {
             const paragraphInfo = getValueAtPath(this.element, '["p:txBody"][0]["a:p"]');
             const isPlaceholderList = this.isPlaceholderListElement();
 
+            const elementDescr = getValueAtPath(this.element, '["p:nvPicPr"][0]["p:cNvPr"][0]["$"]["descr"]');
```

Comment:
I am not really sure about it. That's why I didn't keep the var name specific as well. How can I find this? I referred to https://docs.microsoft.com/en-us/dotnet/api/documentformat.openxml.presentation.nonvisualdrawingproperties.description?view=openxml-2.8.1#documentformat-openxml-presentation-nonvisualdrawingproperties-description

Reply:
I follow the documentation [here](http://officeopenxml.com/drwSp-nvSpPr.php). I usually hook up that specific page of the documentation as well in the PR description for future references. 
<img width="808" alt="image" src="https://user-images.githubusercontent.com/16796013/164448642-4d997fde-ab9a-42af-aad8-64ae1c18abe9.png">

So yea this is for alt text only.

Since we don't have it structured under image or something and its there in the root of the element so changing it to altText will help clarify.

Reply:
Agreed renamed to altText